h2/src/main/org/h2/mvstore/MVStore.java

```diff
@@ -127,7 +127,26 @@ to a map (possibly the metadata map) -
 /**
  * A persistent storage for maps.
  */
-public class MVStore implements AutoCloseable {
+public class MVStore implements AutoCloseable
+{
+    // The following are attribute names (keys) in store header map
+    private static final String HDR_H = "H";
+    private static final String HDR_BLOCK_SIZE = "blockSize";
+    private static final String HDR_FORMAT = "format";
+    private static final String HDR_CREATED = "created";
+    private static final String HDR_FORMAT_READ = "formatRead";
+    private static final String HDR_CHUNK = "chunk";
+    private static final String HDR_BLOCK = "block";
+    private static final String HDR_VERSION = "version";
+    private static final String HDR_CLEAN = "clean";
+    private static final String HDR_FLETCHER = "fletcher";
+
+    // The following are key prefixes used in meta map
+    public static final String META_CHUNK = "chunk."; // + hex chunk id -> serialized chunk metadata
+    public static final String META_NAME = "name.";   // + map's name -> hex map id
+    public static final String META_MAP = "map.";     // + hex map id -> serialized map metadata
+    public static final String META_ROOT = "root.";   // + hex map id -> hex root page "position"
```

Comment:
`MVStore` class is used directly by end users, I think it should not have internal constants with `public` visibility.

Perhaps they can have default visibility and test cases can have own copies of these constant, or they can be moved into some other class or interface with constants.

Reply:
Also they are referenced from the `MVStore` class in inconsistent way: some references use only the name, others have a `MVStore.` prefix. It complicates reading a little bit.

Reply:
I'll fix those three instances.
As far as visibility and exposure goes, what's the harm exposing strings? Yes, we the can be moved into another class/interface, but still have to be public.

Reply:
I don't like strange items in the javadoc, autocompletion list, and other places. Such constants look like something useful for the end user, but they aren't.

Of course, it's not a big deal.

Reply:
Of course, it's not something useful for the end user, but let developers segregate usage of string "chunk" in the store header, from usages in chunk metadata, from usages in meta map keys (and later change them into different strings?).
I am with you on a namespace pollution issue, so how about DataUtils, which already has the whole bunch?

Reply:
Yes, I also thought about that class.